.gitignore

```diff
@@ -35,4 +35,5 @@ aura-inspector-tests/src/main/webapp/dist/
 coverage/
 logs/
 .vscode/settings.json
+.history
```

Comment:
Who's adding a .history folder?

Reply:
This is VScode plugin for file local history (similar to the local history feature on Intellij)
I added because I used it and as this is a well used plugin, I prefer to add it here to avoid adding into a commit this kind of files.